### PR TITLE
Documentation 

### PR DESCRIPTION
**Pull request by alexandra-plana**
main page for documentation for each milestone. Will be accessed through the board for the milestone. This page will contain the different documentation topics (title) and the article preview. When you click on the article a text editor will be open (intention to use Quill). Will have to work on the SSE and bookmarks too. Create, delete update is set up but no buttons have been created yet. Intention to add search functionality to find the articles by their title.🍪

### CODE REVIEW

**Review by lexarisa — APPROVED** [review]
goodjobbbb <3

File: server/src/app.ts [hunk 1]
```diff
@@ -22,5 +23,6 @@ app.use(userRouter);
 app.use(feedRouter);
 app.use(projectRouter);
 app.use(milestoneRouter);
+app.use(documentRouter);
```
Comment (thaistcosta) [inline server/src/app.ts:26]: I'm not sure... but it seems that the article router missing...

**Review by thaistcosta — APPROVED** [review]
Chica !!! Uaaaau ! 

Tu es increiblee  ❤️ ❤️  

Now we have Docs !!!!

Muchas graciasss 🍪

File: server/src/app.ts (hunk 1 above)
Comment (alexandra-plana) [reply server/src/app.ts:26]: it absolutely is, i'll add it right away

**Pull request merged**